frontend/src/main/scala/akkaviz/frontend/vis/package.scala

```diff
@@ -0,0 +1,249 @@
+
+  @js.native
+  @JSName("vis.DataSet")
+  class DataSet[T](data: js.Array[T] = js.Array[T](), options: js.UndefOr[DataSetOptions] = js.native) extends js.Any {
+
+    def add(d: T): js.Array[String] = js.native
+
+    def add(d: js.Array[T]): js.Array[String] = js.native
+
+    def remove(d: String): js.Array[String] = js.native
+
+    def remove(d: js.Array[String]): js.Array[String] = js.native
+
+    def get(d: String): T = js.native
+
+    def get(d: js.Array[String]): js.Array[T] = js.native
+
+    def setOptions(o: DataSetOptions): Unit = js.native
+
+  }
+
+  @js.native
+  trait NetworkData extends js.Any {
+    var nodes: js.Array[Node] | DataSet[Node] = js.native
+    var edges: js.Array[Edge] | DataSet[Edge] = js.native
+  }
+
+  object NetworkData {
+    def apply(nodes: js.Array[Node] | DataSet[Node], edges: js.Array[Edge] | DataSet[Edge]): NetworkData = {
+      js.Dynamic.literal(nodes = nodes.asInstanceOf[js.Any], edges = edges.asInstanceOf[js.Any]).asInstanceOf[vis.NetworkData]
+    }
+  }
+
+  @js.native
+  trait NetworkOptions extends js.Any
+
+  object NetworkOptions {
+    //TODO: define typesafe builder
+    def apply(): NetworkOptions = {
+      js.Dynamic.literal().asInstanceOf[NetworkOptions]
+    }
+  }
+
+  @js.native
+  trait Coords extends js.Any {
+    val x: Double
+    val y: Double
+  }
+
+  object Coords {
+    def apply(x: Double, y: Double): Coords = {
+      js.Dynamic.literal(x = x, y = y).asInstanceOf[Coords]
+    }
+  }
+
+  @js.native
+  trait NetworkMoveToOptions extends js.Any {
+    var position: Coords = js.native
+    var scale: Double = js.native
+    var offset: Coords = js.native
+    //var animation
+  }
+
+  @js.native
+  trait NetworkFitOptions extends NetworkMoveToOptions {
+  }
+
+  @js.native
+  trait NetworkFocusOptions extends NetworkMoveToOptions {
+    var locked: Boolean = js.native
+  }
+
+  @js.native
+  @JSName("vis.Network")
+  class Network(element: Element, data: NetworkData, options: NetworkOptions) extends js.Any {
+
+    //Global methods for the network
+    def destroy(): Unit = js.native
+
+    def setData(data: NetworkData): Unit = js.native
+
+    def setOptions(options: NetworkOptions): Unit = js.native
+
+    def on(event: String, fn: js.Function): Unit = js.native
+
+    def off(event: String, fn: js.Function): Unit = js.native
+
+    def once(event: String, fn: js.Function): Unit = js.native
+
+    //Methods related to the canvas
+    def canvasToDOM(coords: Coords): Coords = js.native
+
+    def DOMtoCanvas(coords: Coords): Coords = js.native
+
+    def redraw(): Unit = js.native
+
+    def setSize(width: String, height: String): Unit = js.native
+
+    //Methods to get information on nodes and edges.
+
+    //Physics methods to control when the simulation should run.
+
+    //Selection methods for nodes and edges.
+
+    //Methods to control the viewport for zoom and animation.
+    def getScale(): Double = js.native
+
+    def getViewPosition(): Coords = js.native
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Do not prefix getters with 'get'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2159111834/issues/source?bid=3121282&fileBranchId=3158877#l146)